build.sbt

```diff
@@ -164,7 +167,10 @@ lazy val test = crossProject(JSPlatform, JVMPlatform)
   )
 
 lazy val testJVM = test.jvm.settings(dottySettings)
-lazy val testJS  = test.js
+lazy val testJS = test.js
+  .settings(
```

Comment:
I don't think we want to add this. This will force everyone using ZIO Test for ScalaJS to bring in the `scala-java-time` dependency instead of bringing their own. Whether we should take a different approach to our zero dependencies policy here is a good question, but for this PR let's just fix the issue so contributors don't have an issue compiling the project.

Reply:
Fair enough about the dependency. Alternatively (or in addition) to this PR, I could make a note of this in the contributors README. What do you think?

Reply:
Well I think once we merge this it should just work for new contributors working on the project, right? The need to bring your own time dependency should only apply to end users depending on ZIO Test.

Reply:
Correct, the dependencies fixed the errors I reported yesterday in my env.
For end users, the documentation about bringing your own dependency is [here](https://github.com/zio/zio/blob/master/docs/interop/javascript.md)

Reply:
Yes, so let's just add the dependency to the streams tests and merge this.